portfolio/src/main/webapp/quiz.js

```diff
@@ -0,0 +1,58 @@
+
+
+function makeQuiz(){
+    questions.forEach((currentQuestion,questionNumber)=>{
+        output.push(
+            `<input type="checkbox" id="question${questionNumber}" name="question${questionNumber}" 
+            value="yes">
+            <label for="question${questionNumber}">${currentQuestion}</label></br>`);
+    });
+    quizContainer.innerHTML=output.join('');
+}
+function showResults(){
```

Comment:
Format this file with prettier.

You can install it with `npm -g install prettier`. Then run it with `prettier --write portfolio/src/main/webapp/quiz.js`.

Or you can copy and paste it into the online playground: https://prettier.io/playground/

Reply:
Ok!